metrics/web/lib/dashboard/presentation/state/project_metrics_notifier.dart

```diff
@@ -198,7 +211,22 @@ class ProjectMetricsNotifier extends ChangeNotifier {
   }
 
   /// Sets the [selectedProjectGroup] to project group with the given [id].
+  ///
+  /// Updates the current page parameters with the selected project id
+  /// or an empty string if the id is `null`.
   void selectProjectGroup(String id) {
+    _updateSelectedProjectGroup(id);
+
+    final projectGroupId = _selectedProjectGroup?.id ?? '';
+
+    _updateCurrentPageParameters(projectGroupId: projectGroupId);
+  }
+
+  /// Sets the [selectedProjectGroup] to project group with the given [id].
+  ///
+  /// Does nothing if the [_projectGroupDropdownItems] does not contain
+  /// the project group with the given [id].
+  void _updateSelectedProjectGroup(String id) {
```

Comment:
🔖 
```suggestion
  void _selectProjectGroup(String id) {
```
Would it be a bit more clear?